tests/test_analyzer.py

```diff
@@ -110,12 +114,17 @@ def test_correlation_helper_for_group_with_one_row_and_length(self):
 
 
 
-    def test_that_correlation_helper_works_for_data_with_the_same_label(self):
-        with warnings.catch_warnings():
+    def test_that_correlation_helper_works_for_data_with_the_same_human_score(self):
+        # this test should raise UserWarning because the determinant is very close to
+        # zero. It also raises Runtime warning because
+        # variance of human scores is 0.
+        with warnings.catch_warnings(record=True) as w:
             warnings.filterwarnings('ignore', category=RuntimeWarning)
             retval = Analyzer.correlation_helper(self.df_features_same_score, 'sc1', 'group')
             assert_equal(retval[0].isnull().values.sum(), 3)
             assert_equal(retval[1].isnull().values.sum(), 3)
+            assert issubclass(w[-1].category, UserWarning)
```

Comment:
```suggestion
            assert issubclass(warning_list[-1].category, UserWarning)
```
Same comment here for `[0]` vs `[1]`.